ggml/src/ggml-metal/ggml-metal-impl.h

```diff
@@ -519,6 +519,7 @@ typedef struct {
     int64_t  n_group;
     int64_t  n_seq_tokens;
     int64_t  n_seqs;
+    int64_t  s_off;
```

Comment:
I'm not sure this is actually necessary. I did this when initially trying to emulate the CUDA kernel better which passes this as an arg. It does seem like it might be _slightly_ faster to avoid computing this in the kernel, though that could be offset by the latency of an additional `int64_t` being passed to the device?